doctr/models/predictor/pytorch.py

```diff
@@ -47,8 +47,8 @@ def __init__(
     ) -> None:
 
         super().__init__()
-        self.det_predictor = det_predictor.eval()  # type: ignore[attr-defined]
-        self.reco_predictor = reco_predictor.eval()  # type: ignore[attr-defined]
+        self.det_predictor = det_predictor  # type: ignore[attr-defined]
+        self.reco_predictor = reco_predictor  # type: ignore[attr-defined]
```

Comment:
Is this an error? We should keep this to ensure that predictor are used for inference

Reply:
Can you specify the command you run to get this error?

It will only run when Pytorch backend is enabled (set `USE_TORCH='1'` as env variable before running python)
The `.eval` method is inherited from `torch.nn.Module` so this part shouldn't be changed